logstash-core/lib/logstash/environment.rb

```diff
@@ -4,6 +4,7 @@
 require "logstash/settings"
 require "socket"
 require "stud/temporary"
+require "logstash/java_integration"
```

Comment:
Can we move that require with the other `logstash` one? So they are all together.